### PR TITLE
cask aliases update

### LINKED ISSUE #1866: caskl return error 1
### Description
`caskl` tries to run new way to list casks, but an error code is returned.

### Expected behavior
Listing of installed casks

### Actual behavior
```
Error: Calling brew cask list is deprecated! Use brew list --cask instead.
-> [1]
```

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

### Versions

  - Prezto commit: 0a405494cae54e208ff550c1ba83380ed115410a
  - ZSH version: zsh 5.7.1
  - OS information: macOS 10.15.6

### PR DESCRIPTION
**Pull request by Rawmanski**
Please be sure to check out our [contributing guidelines](https://github.com/sorin-ionescu/prezto/blob/master/CONTRIBUTING.md)
before submitting your pull request.

Fixes #1866 

## Proposed Changes

  - deprecated cask aliases
  - 
  -

### CODE REVIEW

**Comment by Rawmanski** [comment]
Updated cask aliases, because old ones are deprecated. Hence `brew cask install` isn't working anymore, instead use `brew install --cask`.

File: modules/homebrew/init.zsh [hunk 1]
```diff
@@ -39,14 +39,11 @@ alias brewu='brew upgrade'
...
-alias casko='brew cask outdated'
-alias casks='hb_deprecated brew cask search'
-alias caskx='brew cask uninstall'
+alias caski='brew install --cask'
+alias caskl='brew list --cask'
+alias casko='brew outdated --cask'
+alias casks='brew search --cask'
+alias caskx='brew uninstall --cask'
 
 function hb_deprecated {
```
Comment (jeffwidman) [inline modules/homebrew/init.zsh:48]: Shouldn't this also be removed since it's no longer used??

Comment (indrajitr) [reply modules/homebrew/init.zsh:48]: Well, kinda :) Actually, we shouldn't be abruptly dropping alias without a migration warning. This is in my backlog. Will cherry-pick the commit and apply the necessary changes.

Comment (jeffwidman) [reply modules/homebrew/init.zsh:48]: While I am not opposed to a migration warning, I am also _completely_ fine with dropping aliases without a migration warning. I _really_ don't want the expectation to be that we have to do a formal migration for everything, especially more minor things like simple aliases.

So far we've kept the focus that this is a low-maintenance thin shell wrapper targeting power users, so I think it's fine to have an assumption of "read the commit log when you upgrade" so as long as we clearly document backwards breaking changes it in the commit title, then I think that's sufficient. We aren't that active of a project, so as long as you update once or twice a year it should take < 10 mins to skim the commit log.

Again, I am totally not opposed if you want to go to the additional work of adding migration warnings, as long as it doesn't become an expectation/requirement except for core functionality changes (and even then, I'm often still fine with "the migration message is the commit history").

Comment (indrajitr) [reply modules/homebrew/init.zsh:48]: Okay @jeffwidman, compelling arguments! 👍🏽

**Comment by indrajitr** [comment]
Created #1905 cherry-picking commits from @Rawmanski.

**Comment by indrajitr** [comment]
Closed via #1905

**Pull request closed without merging**